bibliopixel/project/project.py

```diff
@@ -76,23 +82,42 @@ def resolve_aliases(name):
 
     if not (driver or drivers):
         raise ValueError(
-            'The project has neither a "driver" nor a "drivers" sections')
+            'The project has neither a "driver" nor a "drivers" section')
 
     extend_path(path)
     maker = data_maker.Maker(**(maker or {}))
     make_object = functools.partial(importer.make_object, maker=maker)
 
-    builder = MultiMapBuilder(make_object)
-    driver_objects = builder.make_drivers(driver, drivers)
+    driver_objects = make_drivers(driver, drivers, make_object)
+
+    gen_coord_map = layout.pop('gen_coord_map', None)
+
+    if 'coord_map' not in layout and gen_coord_map:
+        typename = layout['typename']
+        if typename not in MULTI_HANDLERS:
+            raise ValueError('There is currently no available multi-map builder for {}'.format(typename))
+        gen_multi = MULTI_HANDLERS[typename]
+        if isinstance(gen_coord_map, dict):
```

Comment:
You can also apply transforms on a whole map after built...
So, if you passed a dict it would be:
```
"gen_coord_map": {
    "config": [ ... list of row dicts or lists ... ],
    "rotation": 3,
    "y_flip": true
}
```